app/index.html

```diff
@@ -7,9 +7,13 @@
   <link rel="stylesheet" href="emoji/emoji-20px.css">
   <link rel="stylesheet" href="emoji/emoji-45px.css">
   <link rel="stylesheet" href="styles/main.css">
-  <!-- Once we change our actual l10n framework to l10n.js (or better, l20n.js),
-       we need to merge fxosrate strings to our new l10n files -->
-  <link rel="resource" type="application/l10n" href="components/fxosRate/locales/locales.ini" />
```

Comment:
You are removing the `fxosrate` files, but you have not added the strings to the new `properties` file, right?